.github/ISSUE_TEMPLATE/epic_release.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Epic Release
+about: Epic Release Checklist
+title: 'Epic Release: [epic name]'
+labels: epic-release-activities
+assignees: ''
+
+---
+
+The following is a checklist for new epics or features acceptance for Spartacus, based on [our Definition of Done](https://sap.github.io/spartacus-docs/definition-of-done/)
+
+## General
+
+- [ ] Epic has been tested on a production-like server using Spartacus installation script
+- [ ] Epic has been tested in all supported browsers and devices (Check list on DoD page)
+- [ ] Epic has sufficient end to end (cypress) test coverage
+- [ ] Schematics code deprecations have been provided and tested
+- [ ] Epic complies with [Security best practices](https://sap.github.io/spartacus-docs/security-best-practices/)
+- [ ] Epic complies with [Accessibility best practices](https://sap.github.io/spartacus-docs/a11y-best-practices/)
+- [ ] Epic CSS supports directionality (if required)
+- [ ] Epic code scans have been conducted (sonar, whitesource, checkmarx)
+
+## Audits/reviews
+
+- [ ] Architecture review of the epic (with @tobi-or-not-tobi)
+- [ ] Security and threat modeling audit/review with the Security team (main contact is @giancorderoortiz)
```

Comment:
Should we always start with @giancorderoortiz here? For our second increment, we are trying to organize a session with the central commerce team (Michael Riedel) like we did for the first iteration.

Reply:
Yes, I think we should rely on teams rather than people. Thanks for the feedback.